lib/frontman/app.rb

```diff
@@ -127,6 +127,19 @@ def add_asset_pipeline(
       )
     end
 
+    sig { params(file_path: String).void }
+    def import_config(file_path)
+      if !File.exist?(file_path) && !file_path.end_with?(".rb")
+        other_path = "#{file_path}.rb"
+
+        if File.exist?(other_path)
+          file_path = other_path
+        end
```

Comment:
```suggestion
        return import_config("#{file_path}.rb")
```

I think we can simplify this method this way. If we append the `.rb`, the first check will fail, and it will try to read the non-existing file, which would happen in the other case too. Wdyt?